common/blobtest/helpers.go

```diff
@@ -2,14 +2,24 @@ package blobtest
 
 import (
 	"crypto/rand"
+	"math/big"
 	"testing"
 
 	"github.com/attestantio/go-eth2-client/spec/deneb"
 	"github.com/attestantio/go-eth2-client/spec/phase0"
+	gokzg4844 "github.com/crate-crypto/go-kzg-4844"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/stretchr/testify/require"
 )
 
+// BLS12-381 field modulus (as a big-endian byte array)
+// This is the prime field modulus used by BLS12-381
+// In decimal: 52435875175126190479447740508185965837690552500527637822603658699938581184513
```

Comment:
Super small, but might make sense to name this subgroup order or something since there are multiple field moduli involved in BLS12-381 (naming reference: https://hackmd.io/@benjaminion/bls12-381)

Reply:
Fixed in 4b122092a8128970d911bbd1faa29114faaef942 by just using the library's exported version of this constant and using the correct name in a comment.